client/app/telecom/telephony/alias/configuration/feature/redirect/telecom-telephony-alias-configuration-redirect.html

```diff
@@ -0,0 +1,59 @@
+<oui-spinner data-ng-if="$ctrl.loading"></oui-spinner>
+
+<div data-ng-if="!$ctrl.loading">
+    <div class="mb-5">
+        <h3 class="oui-heading_underline"
+            data-translate="telephony_alias_config_redirect_incoming_calls_heading"></h3>
+        <dl>
+            <dt data-translate="telephony_alias_config_redirect_line_to_redirect"></dt>
+            <dd>
+                <span data-ng-if="!$ctrl.newDestination"
```

Comment:
Otherwise you could simply apply the `ngIf` directive to the `<dd>` element (need twice).